controllers/cartController.js

```diff
@@ -0,0 +1,43 @@
+const cartService = require("../services/cartService");
+const { catchAsync } = require("../middleware/error");
+
+const getCartList = catchAsync(async (req, res) => {
+  try {
+    const userId = req.user.id;
+    const cartsData = await cartService.getCartList(userId);
+
+    return res.status(200).json({ data: cartsData });
+  } catch (err) {
+    return res.status(err.statusCode || 400).json({ message: err.message });
+  }
+});
+
+const updateCartQuantity = catchAsync(async (req, res) => {
+  try {
+    const userId = req.user.id;
+    const { quantity } = req.body;
+    const cartId = req.params.cartId;
+
+    const cart = await cartService.updateCartQuantity(userId, quantity, cartId);
+
+    return res.status(200).json({ cart });
+  } catch (err) {
+    return res.status(err.statusCode || 400).json({ message: err.message });
+  }
+});
+
+const deleteCartItems = catchAsync(async (req, res) => {
+  try {
+    const { cartId } = req.query;
+    await cartService.deleteCartItems(cartId);
+    return res.status(204).json({ cartId });
```

Comment:
```suggestion
    return res.status(204).send();
```
status code 204의 상태 메세지는 `No content`입니다. http에 따르면 body 값은 method 혹은 status에 따라 body가 있을 수도, 없을 수도 있습니다. express에서 204 status code를 활용하면 body에 어떠한 값을 담아서 반환하지 않기 때문에 send() 메소드를 활용하여 response만 해도 됩니다!

궁금하시면 실제로 postman으로 테스트 해보셔도 좋습니다 :)